### PR TITLE
Add plugins for osconfig repository

### PR DESCRIPTION
**Pull request by rofuentes**
Just adding additional plugin for our repository. Job config is in #13

### CODE REVIEW

File: prow/plugins.yaml [hunk 1]
```diff
@@ -39,3 +39,7 @@ plugins:
   - shrug
   - wip
   - yuks
+
+  GoogleCloudPlatform/osconfig:
+  - config-updater
```
Comment (krzyzacy) [inline prow/plugins.yaml]: config all lives in this oss-test-infra repo, you don't need config-updater in your repo

Comment (rofuentes) [reply prow/plugins.yaml]: Removed.

**Comment by krzyzacy** [comment]
(need to also set up tide in this repo)

**Comment by krzyzacy** [comment]
hummmmmmmmmmmm also looks like config-updater didn't work as expected

**Pull request merged**